docs/source/additional/release.rst

```diff
@@ -31,6 +31,23 @@ This is an overview of the main changes, please have a look at the changelog of
 - `backend changelog <https://github.com/Substra/substra-backend/blob/main/CHANGELOG.md>`__
 - `orchestrator changelog <https://github.com/Substra/orchestrator/blob/main/CHANGELOG.md>`__
 
+Substra 0.38.0 --- 2024-06-13
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**All**
+
+- The release 0.38.0 focuses on improving the security and limit the rights of the differents Substra pods when deployed on a cluster. The following changes have been made:
+
+  - Ressources requests and limits are now set as default for the backend and frontend.
+  - Network policies have been added to limit the communication between the different pods.
+  - Update the different base docker images of the Susbtra components and the the user's ones to limit their vulnerabilities.
+
+- The profiling of functions have been reworked to take into accound the ``builiding`` step and the ``download_function`` step.
+
+**Backend**
```

Comment:
This is not only backend, but also orchestrator (or the title sounds confusing to me)

Reply:
ah my bad I thought it was only on backend, changed it